javascript-package/tst/index.test.js

```diff
@@ -215,23 +215,17 @@ describe("client", () => {
             },
           })
         );
-        const mockLeague = LEAGUES.USLC;
```

Comment:
It turns out the intersection of all parameters for all methods is only the seasonName parameter. The jest failure revealed I was passing invalid parameters.